src/assets/content/portfolio.json

```diff
@@ -2,7 +2,7 @@
   "portfolioItems": [
     {
       "id": "item1",
-      "imgSrc": "/naszownik.jpeg",
+      "imgSrc": "/assets/images/naszownik.jpeg",
```

Comment:
About images - there can be in something like "assets/images" but dont know if Astro doesnt prefer to put it in src for optimization of these files. Please chceck that in docs for confirmation :)